### PR TITLE
update files for networkAware plugins

### LINKED ISSUE #582: [Network-Aware-Scheduling] Extra deployment files are needed
The available documentation + deployment files seem to be insufficient for deploying the Network-aware scheduling plugins since developers have been facing issues. 

A few extra deployment files would help developers how to set up everything properly. 

I will be working on these modifications.
/assign

### PR DESCRIPTION
**Pull request by jpedro1992**
#### What type of PR is this?

Updating deployment files for the Network Aware plugins to ease its deployment. 

#### What this PR does / why we need it:

Extra files available to help developers/contributors to deploy the Network Aware plugins. 

#### Which issue(s) this PR fixes:

Fixes #582

### CODE REVIEW

File: manifests/appgroup/deploy-sig-scheduling-controller-and-scheduler.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    metadata:
+      labels:
+        component: scheduler
+        tier: control-plane
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: "master.n1" # Modify to your master node name! Unless you populate all nodes with sched-cc.yaml
+      containers:
+        - image: registry.k8s.io/scheduler-plugins/kube-scheduler:v0.25.7
+          #imagePullPolicy: Never
```
Comment (Tal-or) [inline manifests/appgroup/deploy-sig-scheduling-controller-and-scheduler.yaml:117]: Please uncomment or remove if not needed

File: manifests/appgroup/deploy-sig-scheduling-controller-and-scheduler.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          command:
+            - /bin/kube-scheduler
+            - --authentication-kubeconfig=/etc/kubernetes/scheduler.conf
+            - --authorization-kubeconfig=/etc/kubernetes/scheduler.conf
+            - --config=/etc/kubernetes/sched-cc.yaml
+            - -v=9
```
Comment (Tal-or) [inline manifests/appgroup/deploy-sig-scheduling-controller-and-scheduler.yaml:123]: Is it has to be so verbose by default?

File: manifests/networktopology/deploy-sig-scheduling-controller-and-scheduler.yaml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    metadata:
+      labels:
+        component: scheduler
+        tier: control-plane
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: "master.n1" # Modify to your master node name! Unless you populate all nodes with sched-cc.yaml
+      containers:
+        - image: registry.k8s.io/scheduler-plugins/kube-scheduler:v0.25.7
+          #imagePullPolicy: Never
```
Comment (Tal-or) [inline manifests/networktopology/deploy-sig-scheduling-controller-and-scheduler.yaml:117]: ditto

File: manifests/appgroup/deploy-sig-scheduling-controller-and-scheduler.yaml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+      component: scheduler
+      tier: control-plane
+  template:
+    metadata:
+      labels:
+        component: scheduler
+        tier: control-plane
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: "master.n1" # Modify to your master node name! Unless you populate all nodes with sched-cc.yaml
```
Comment (Tal-or) [inline manifests/appgroup/deploy-sig-scheduling-controller-and-scheduler.yaml:114]: if you want it to be running only on master you can use:
`node-role.kubernetes.io/master: ""`
instead, without having the need to ask the user to modify it.

**Review by Tal-or — COMMENTED** [review]
Thank you for your PR, I added couple of comments.

**Comment by jpedro1992** [comment]
Thank you @Tal-or for your comments. I updated the deployment files.

**Comment by ffromani** [comment]
@jpedro1992 do we need a release note? 
I'll review myself shortly.

**Comment by Huang-Wei** [comment]
> @jpedro1992 do we need a release note?

I think it's fine to leave release notes empty.

/lgtm
/approve
/release-note-none

**Pull request merged**